src/models/time_series/realmlp/realmlp_predictor.py

```
@@ -0,0 +1,817 @@
+                has_cat = hasattr(self.model, "cat_embedding") and getattr(self.model, "cat_embedding") is not None
+                if has_cat:
+                    dummy_cat = torch.zeros((input_tensor.shape[0],), dtype=torch.bfloat16, device=self.device)
+                    predictions_example = self.model(input_tensor, dummy_cat).detach().cpu().numpy()
+                else:
+                    predictions_example = self.model(input_tensor).detach().cpu().numpy()
+            signature = mlflow.models.infer_signature(input_example, predictions_example)
+        finally:
+            try:
+                self.model.to(self.device)
+            except Exception:
+                pass
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)